gcc/rust/hir/tree/rust-hir-full-decls.h

```diff
@@ -29,14 +29,9 @@ class TokenTree;
 class MacroMatch;
 class Token;
 struct Literal;
-class DelimTokenTree;
 class PathSegment;
 class SimplePathSegment;
 class SimplePath;
```

Comment:
Do you mean to remove `SimplePath` here?

Reply:
Everybody else now uses `AST::SimplePath`, correct?

Reply:
yeah, let me try to remove it.

Reply:
LOL, I found this is another miss deletion. I already remove the SimplePath definition but decl.